brave/brave-core
# Updated brave help bubble UI

fix https://github.com/brave/brave-browser/issues/38237

<img width="467" alt="Screenshot 2024-05-14 at 09 56 45" src="https://github.com/brave/brave-core/assets/6786187/94c66581-4c45-4909-aaa3-e89f45ce8bfb">
<img width="445" alt="Screenshot 2024-05-14 at 09 57 24" src="https://github.com/brave/brave-core/assets/6786187/51469a1f-c4bb-4921-bc47-f18b930cd9db">


Resolves



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch brave with clean profile
2. Load any url that has trackers such as bbc.com
3. Check bubble only shows the number of blocked count
